components/Button/index.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+
+import Button from '.';
+
+describe('Button', () => {
+  it('renders a button', () => {
+    const props = {
+      type: 'reset',
+      size: 'small',
+      dark: true,
+      dataTestId: 'reset_button',
+      children: 'reset',
+    };
+
+    render(<Button {...props} />);
+
+    const button = screen.getByTestId(props.dataTestId);
+
+    expect(button).toBeVisible();
+    expect(button).toHaveTextContent(props.children);
+    expect(button).toHaveAttribute('type', props.type);
+  });
```

Comment:
Should we also add the cases to check the classes names for sizing and color? 💭

Reply:
sure, added on d129258d877c1e23ba31313cc0390ad9627eb192